### PR TITLE
feat: apollo subscriptions

### PR DESCRIPTION
**Pull request by momentiris**
**Ticket:** [#45](https://trello.com/c/AQLhE1YY/45-apollo-subscriptions)
**Intent:**
Provide graphql route for the consent request approval.
Client subscribes to this route and gets response once client approved myskills in mydata.

- [x] I've written tests for the code. 

### All of the following commands should pass.

- [] `npm test`
- [ ] `npm run integration`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,9 +31,13 @@
   "homepage": "https://github.com/Iteam1337/myskills-api#readme",
   "dependencies": {
     "@mydata/client": "^0.22.3",
+    "apollo-cache-inmemory": "^1.5.1",
+    "apollo-client": "^2.5.1",
     "apollo-codegen": "^0.20.2",
     "apollo-datasource-rest": "^0.3.2",
     "apollo-errors": "^1.9.0",
+    "apollo-link-http": "^1.5.14",
```
Comment (lundeeell) [inline package.json]: Where do we use apollo-link-http?

File: lib/server.ts [hunk 2]
```diff
@@ -76,16 +78,18 @@ server.applyMiddleware({
...
  */
 
 const port = config.SERVER_PORT
 
 export const appIsReady: Promise<Boolean> = new Promise(resolve =>
-  app.listen(port, async () => {
-    await mydataConnect()
-
+  httpServer.listen(port, () => {
+    mydataConnect()
```
Comment (lundeeell) [inline lib/server.ts]: Why remove the await?

Comment (momentiris) [reply lib/server.ts]: Ops. Unintentional! Perhaps the result of some earlier problem investigation.. i fix!

File: package.json (hunk 1 above)
Comment (momentiris) [reply package.json]: We don’t. Saw this before as well but got caught in other stuff and forgot abt it. Good catch. I fix!

**Pull request merged**